pkgs/development/ocaml-modules/ocaml-gettext/camomile.nix

```diff
@@ -0,0 +1,16 @@
+{ buildDunePackage, ocaml_gettext, camomile, ounit }:
+
+buildDunePackage {
+  pname = "gettext-camomile";
+  inherit (ocaml_gettext) src version useDune2;
+
+  propagatedBuildInputs = [ camomile ocaml_gettext ];
+
+  doCheck = true;
+  checkInputs = [ ounit ];
```

Comment:
Doesn't need fileutils?!

Reply:
Probably, but it comes along with `ocaml_gettext`…

Reply:
As `filutils` is explicitly listed there: https://github.com/gildor478/ocaml-gettext/blob/v0.4.2/test/test-camomile/dune#L7, I’ve just added it to the `checkInputs`. Thanks.